.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+name: Release
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v4
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: '27'
+          rebar3-version: '3.24'
+      - name: Publish to hex.pm
+        env:
+          HEX_API_KEY: ${{ secrets.HEX_API_KEY }}
+        run: rebar3 hex publish -r hexpm --yes
```

Comment:
I'd put a step for `rebar3 compile` before. It almost never a problem, surely won't in this repo, but if you had ASN1 or C code compilation steps, hex won't build them and the published package would be invalid.

Just a good practice, something like.
```suggestion
      - run: rebar3 compile
      - name: Publish to hex.pm
        env:
          HEX_API_KEY: ${{ secrets.HEX_API_KEY }}
        run: rebar3 hex publish -r hexpm --yes
```

Reply:
It is commented out in our standard templates. Should we make it a default?
https://github.com/dnsimple/dnsimple-engineering/blob/main/templates/erlang/release.yml

Reply:
Yeah, by default it should be enabled. As said, it'll only be an issue with ASN1 or NIFs, but still, I'd vote to keep it enabled by default as a safe template.